src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Management.Resources;
```

Comment:
And I use this version already. Here is reference from Network.Tests.csproj
    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />